src/error.rs

```diff
@@ -45,6 +47,7 @@ impl Error {
             Error::CrateConfig { message: _ } => {
                 "There was a crate configuration error. Details:\n\n"
             }
+            Error::DirNotFound { message: _ } => "Unable to find the directory\n\n",
```

Comment:
As I understand it, this works as a default message. I considered using a more specific `PkgNotFound` error, but if everyone made specific error like that, it would be a mess to work with. This feels easier to use.